sycl/source/detail/usm/usm_impl.cpp

```diff
@@ -41,6 +41,20 @@ namespace usm {
 void *alignedAllocHost(size_t Alignment, size_t Size, const context &Ctxt,
                        alloc Kind, const property_list &PropList,
                        const detail::code_location &CodeLoc) {
+  // if no device in context has spect::usm_host_allocations then raise
+  // feature_not_supported exception
+  bool raiseUsmException = true;
```

Comment:
The variable names introduced are not consistent with the surrounding convention